src/AdvicesContainer.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from "react";
+import "./index.css";
+import Button from "./Button";
+import Advice from "./Advice";
+import pauseMb from "./images/pattern-divider-mobile.svg";
+import pauseDt from "./images/pattern-divider-desktop.svg";
+const url = "https://api.adviceslip.com/advice";
+
+const AdvicesContainer = () => {
+	const [advices, setAdvices] = useState("");
```

Comment:
It is always best to keep the state consistent - so, if you expect an array or object as a non-empty state, the same type, but not populated should be the initial state.